config/samples/config.yaml

```diff
@@ -1,26 +1,69 @@
 clusterName: "example-cluster"
 timeIntervalSec: 60
 repoCredentials:
-  "key1":
-    HTTPCredential:
-      Username: "exampleUser"
-      Password:
-        name: "http-creds"
-        key: "password"
-  "key2":
-    SSHCredential:
-      SSHKey:
-        name: "ssh-creds"
-        key: "sshKey"
-  "key3":
-    TLS:
-      insecureSkipVerify: false
-      CACertSecret:
-        name: "ca-cert-secret"
-        key: "ca.crt"
-      certSecret:
-        name: "cert-secret"
-        key: "tls.crt"
-      KeySecret:
-        name: "key-secret"
-        key: "tls.key"
+  - url: "github.com/rustyTest/testprivateRepo"
+    credential:
+      HTTPCredential:
```

Comment:
Can we update all of these `json` fields to following?

`httpCredential`
`username`
`password`
`sshCredential`
`sshKey`
`tls`
`caCertSecret`
`keySecret`

Follow the practice of Kubernetes.

https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/autoscaling/v1/types.go#L36
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/networking/v1/types.go#L301

Reply:
Addressed

Reply:
Yeah pushed just now @whynowy